1st draft on docs about manta & calamari (#10)
Manta-Network/docs: docs/Introduction.md

## Our Mission

We at Manta Network believe that DeFi, for now, has its most value in permissionless and anti-surveillance. In contrast, current services in DeFi have their deficiency in privacy, scalability, usability, and liquidity. **Under the current landscape, only when a stable coin that fully supports anonymity and cross-chain interoperability and the relevant DeFi protocol being created can the tremendous value of DeFi be released and deployed, and it is at this point that DeFi will have the chance to overtake the traditional financial industry. That's the reason why we build Manta Network.**

## Manta Network Overview

Manta's underlying logic is to mint private coins/tokens with stable coins and other base tokens on the contract at a minting value ratio of 1:1. Apart from that, a decentralized private swap platform can support the exchange of these privacy tokens.

> ![Manta Overview](/img/manta_overview.png)

Author: I'm writing the other draft about Governance docs, and when I deliver the draft I'll continue with the comments here. Thanks!

> The Manta Network mainly consists of two protocols: **the decentralized private payment protocol** and the **decentralized private token exchange protocol.**

Reviewer: I would combine this paragraph with the one above the image:

```suggestion
The Manta Network is centered around two protocols: **the decentralized private payment protocol** and the **decentralized private token exchange protocol.** Together they enable private versions of coins to be minted at a 1:1 ratio and then privately exchanged on a decentralized platform. 
```

> ## Products

Reviewer: I don't think you need this header? I think this organization makes sense:
## Manta Network Overview
### Manta Pay: ...
### Manta Swap: ...
### Problems solved...

> ### The Decentralized Anonymous Payment (DAP)

Reviewer 2: ```suggestion
### MantaPay: Privacy Preserving Multi-Asset Decentralized Payment 
```
Please also change DAP to MantaPay in everywhere else.

> Is based on zk-SNARK. It allows users to swap Polkadot and its Parachain tokens with corresponding private tokens, pay with privacy tokens, and redeem base coins from private tokens.

Reviewer: ```suggestion
MantaPay is a protocol based on zk-SNARKs, a blockchain proof technology that keeps user information private. The protocol allows users to swap Polkadot parachain tokens with private versions of those tokens, make transfers with private tokens, and redeem the original coins from private tokens.
```

> ### The Decentralized Anonymous Exchange (DAX)

Reviewer 2: ```suggestion
### MantaSwap: Privacy Preserving Decentralized Exchange
```
Please also change DAX to MantaSwap everywhere else.

> DAX protocol is based on AMM and zk-SNARK. Users can anonymously trade private versions of tokens on the platform between each other. The price formation method of the decentralized private token exchange protocol is consistent with the mainstream AMM. The revenue model of Manta tokens is based on the rebate and redeem of swap usage fees and transaction fees. In the future, the team will also develop products like synthetic asset stable coins based on Manta.

Reviewer: ```suggestion
MantaSwap is a Decentralized EXchange (DEX) protocol is based on an Automated Market Maker (AMM) and zk-SNARK. The protocol creates a platform where users can anonymously trade private versions of tokens. MantaSwap uses a price formation method that is consistent with mainstream AMM, and _the revenue model of Manta tokens is based on the rebate and redeem of swap usage fees and transaction fees._ 
```

I think this should be DEX not DAX?

I'm also not sure what this sentence means: "The revenue model of Manta tokens is based on the rebate and redeem of swap usage fees and transaction fees", so we should clarify.

> We also plan to integrate Manta products into decentralized private lending protocols. Decentralized private lending is a decentralized non-custodial lending platform where users can deposit anonymously and borrow private coins on the platform. Decentralized private synthetic asset protocols and other DeFi services are included in the future roadmap.

Reviewer: ```suggestion
The development team has a roadmap of future product features for MantaSwap, which you can see here [insert link]. Examples of features include synthetic asset stable coins based on Manta, and integrating Manta products into decentralized private lending protocols. Decentralized private lending is a decentralized non-custodial lending platform where users can deposit anonymously and borrow private coins on the platform.
```

This entire paragraph might not be relevant on this doc.

> ## Problems solved and application scenarios in Web3 by Manta

Reviewer: As the comment above suggested, I would add this as a sub-section under ## Manta Network Overview, and rename it "Manta's Role in Web 3.0"

> Although the current distributed ledger has brought a new payment model without the participation of third-party financial institutions, most records of the current cryptocurrency transactions are publicly available due to the transparency of blockchain technology. Public transactions can be viewed by competitors or third-party organizations, which affects business interests. At an individual level, public transactions can be taken advantage of by advertisers and criminals.

Reviewer: ```suggestion
Current distributed ledger technology has brought about a new payment model without the participation of third-party financial institutions, but in most cases transactions records are publicly available. There are many cases where cryptocurrency uses want to keep their transaction records private, such as business protecting their interests from being viewed by competitors and third-party organizations, and individual protecting their activity from advertisers and criminals.
```

> Although privacy-preserving cryptocurrencies such as ZCash exist, the status quo has two significant problems. High price volatility is the first problem, which affects practical use. The lack of scalability is the second problem, which limits use to only simple payments. The exchange of privacy-preserving coins and other cryptocurrencies, including other privacy-preserving coins, is still unsolved — that still relies on centralized exchanges and OTC.

Reviewer: ```suggestion
Although privacy-preserving cryptocurrencies such as ZCash exist, these products have two significant problems. First, high price volatility affects practical use of the coins. And second, the lack of scalability limits use to only simple payments. The exchange of privacy-preserving coins and other cryptocurrencies, including other privacy-preserving coins, is still unsolved — that still relies on centralized exchanges and OTC.
```

Is 'the exchange of privacy-preserving coins in a decentralized manner' a third problem? If it's part of the second, how?

> Manta Network addresses the above two pain points. First, **the decentralized privacy-preserving payment protocol** based on **zk-SNARK** supports Polkadot and Substrate Parachain assets to their corresponding private pairs. In this way, users can use popular assets like stable coins, renBTC, etc., while simultaneously benefiting from the anonymity of privacy coins.

Reviewer: ```suggestion
Manta Network solves both of these pain points. First, **MantaPay** supports Polkadot and Substrate Parachain assets with corresponding private tokens. This allows users to transact in popular assets like stable coins, renBTC, etc., while simultaneously benefiting from the anonymity of privacy coins. Second, **MantaSwap** can realize decentralized transactions between private coins, which is similar to Uniswap and Curve, and ensure transaction addresses' **privacy**.
```

By this point in the doc, Manta Pay and MantaSwap have already been introduced, so the point of this paragraph is just to highlight how Manta does better than any other existing platform.

> Second, **a decentralized privacy-preserving exchange protocol** based on **AMM** and **zk-SNARK** can realize decentralized transactions between private coins, which is similar to Uniswap and Curve, and ensure transaction addresses' **privacy**. Our product is the only platform that can meet user privacy needs, boast anti-surveillance, and ensure private tokens' stability and interoperability. Also, compared to the centralized dark pool, the decentralized version is more credible and secure, as it reduces single points of failure.

Reviewer: ```suggestion
Our product is currently the only platform that can meet user privacy needs, boost anti-surveillance, and ensure stable private tokens with interoperability. Additionally, the decentralized exchange offers more credibility and security compared to centralized pools by reducing single points of failure. 
```

I'm not sure what this paragraph adds, it can probably be combined with the previous one.

If you want to learn more, read our article: [An Introduction of Manta Network](https://mantanetwork.medium.com/an-introduction-of-manta-network-cd2dddf827cb).

## Manta Network's Kusama Chain Project Calamari

> ### About Calamari

Reviewer: I would organize these headers as:
## Calamari Overview
information about calamari at a high level
### MariPay
...
### MariSwap
...

> Calamari, Manta Network's canary net, is the plug-and-play privacy-preservation parachain built to service the Kusama DeFi world. It combines Kusama and zkSNARKs to bring on-chain privacy to transactions and swaps. As part of its suite of products, Calamari Network offers:

Reviewer: ```suggestion
Calamari is Manta Network's canary net. It's where we release new features before they're added to Manta. Calamari serves the Kusama DeFi world by using zkSNARKs to bring on-chain privacy to transactions and swaps. Like Manta, Calamari Network is centered around two main protocols:
```

### MariPay

> Its token-agnostic private payment service. MariPay supports the private transfer of Kusama and its Parachain assets, including major crypto assets supported on Kusama. Users can transact popular purchases like stablecoins and wrapped BTC while simultaneously reaping the benefits of on-chain privacy through ZKP.

Reviewer: ```suggestion
MariPay is a token-agnostic private payment service. MariPay supports the private transfer of Kusama and its Parachain assets, including major crypto assets supported on Kusama. Users can transact popular purchases like stablecoins and wrapped BTC while simultaneously reaping the benefits of on-chain privacy through ZKP.
```

Why is this description so different than MantaPay? Aren't they the same protocol? It seems that the best way to do this would just be to say that MariPay is the same as MantaPay, it will just have newer features. And then you can list the features that MariPay currently has that MantaPay doesn't. Same for MariSwap. If this isn't the case, these docs need to make it a lot more clear what the differences are.

### MariSwap

It's private **AMM-based** based DEX. MariSwap offers users the capability of swaps between parachain assets while preserving the privacy of the user addresses. It also provides never-before-seen features in the industry, such as private liquidity pools.

Calamari will be the only platform on Kusama that can meet user on-chain privacy needs, boost anti-surveillance, and ensure private tokens' decentralization and interoperability.

If you want to learn more, read our article: [Calamari Introduction](/docs/calamari/introduction)

## Community

> To learn more about Manta Network, please follow and join us on our community social platforms below:

Reviewer: ```suggestion
Follow and join our community social platforms!
```
